stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -111,7 +111,7 @@ public enum _RuntimeConfig {
 
   /// When true, uses the TF eager C API, and TF interpreter backend.
   /// Otherwise uses the TF C API, with execution mode set below.
```

Comment:
It'd be nice to add a note here to say that sends/receives for variant and resources tensors are not supported and have undefined behavior when this flag is turned off.

Reply:
Done.